src/Uno.UI.RuntimeTests/Tests/Windows_UI_Xaml_Controls/Given_TextBlock.cs

```diff
@@ -260,5 +260,32 @@ public async Task When_SolidColorBrush_With_Opacity()
 
 			ImageAssert.HasColorInRectangle(bitmap, new System.Drawing.Rectangle(0, 0, bitmap.Width, bitmap.Height), Colors.Red.WithOpacity(.5));
 		}
+
+		[TestMethod]
+		[RunsOnUIThread]
+		public async Task When_TextWrapping_Changed()
+		{
+			var SUT = new TextBlock
+			{
+				TextWrapping = TextWrapping.Wrap,
+				Text = "This is Long Text! This is Long Text! This is Long Text! This is Long Text! This is Long Text! This is Long Text! ",
+			};
+			StackPanel panel = new StackPanel
+			{
+				Width = 100,
+				Children =
+				{
+					SUT
+				}
+			};
+			await UITestHelper.Load(panel);
+			var height1 = SUT.ActualHeight;
+			Console.WriteLine("height1: " + height1);
+			SUT.TextWrapping = TextWrapping.NoWrap;
```

Comment:
![image](https://github.com/unoplatform/uno/assets/122166904/e5047ef8-c83a-43f4-8b68-db05b99660a7)

Reply:
Thanks, it indeed does make sense. Let's let the build run again, we'll take a look at the screenshot tests to see if anything additional shows up.